### ISSUE TITLE
AttributeError: model object has no attribute 'id'

### ISSUE THREAD

**Issue by kierun** [romgar/django-dirtyfields#74]
When running one of my tests (Django 1.9.8 and django-dirtyfields
1.0) I got this weird exception. My `Assembly` model does not have a `id` instead its primary key is called `serial_number`.

Is there an assumption anywhere that all models must have a `id`?

```
Traceback (most recent call last):
  File "/home/usr/repos/intranet/.../tests/models/test_assembly.py", line 257, in test_has_fault_true
    fault.assemblies.add(self.sut)
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/django/db/models/fields/related_descriptors.py", line 843, in add
    self._add_items(self.source_field_name, self.target_field_name, *objs)
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/django/db/models/fields/related_descriptors.py", line 1012, in _add_items
    model=self.model, pk_set=new_ids, using=db)
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/django/dispatch/dispatcher.py", line 192, in send
    response = receiver(signal=self, sender=sender, **named)
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/dirtyfields/dirtyfields.py", line 115, in reset_state
    instance._original_m2m_state = instance._as_dict_m2m()
  File "/home/usr/.virtualenvs/intranet/lib/python2.7/site-packages/dirtyfields/dirtyfields.py", line 74, in _as_dict_m2m
    for f, model in get_m2m_with_model(self.__class__)
AttributeError: 'Assembly' object has no attribute 'id'
```

**Comment by romgar** [comment]
Hi @kierun, some code in `django-dirtyfields` was relying on `id` field, which is wrong as soon as you define a custom primary key in your model.
I have fixed that, it will be available in `1.0.1` release!

**Comment by kierun** [comment]
Thank you very much.
